docs/data_user.md

```diff
@@ -67,7 +67,7 @@ input
 
 ### Output data handling
 
-Output data from the analysis pipeline must be formatted according to [BIDS](https://bids.neuroimaging.io) derivatives. The output directory holds both the intermediate results from the participant-level analysis _and_ the final results from the group-level analysis.
+Output data from the analysis pipeline must be formatted as a typical [BIDS](https://bids.neuroimaging.io) being here `DatasetType: derivatives`. That is required for the shuffling and resampling. The output directory holds both the intermediate results from the participant-level analysis _and_ the final results from the group-level analysis.
```

Comment:
shuffling happens on the input data, so that is a concern for the data rights holder. It also does not require the data to be a derivative.

Let me rephrase this a bit...